app/assets/javascripts/projects/edit_fields.js.coffee

```diff
@@ -65,56 +139,55 @@ $ ->
                  class="field_delete"><i class="fa fa-close slick-delete"></i></a>"""])
       document.getElementById('location').disabled = true
 
-    # Make table sortable
-    $( "tbody" ).sortable();
-
-    # Delete field, enable timestamp/location buttons (NOTE: fid is 0 when the field
-    # hasn't yet been added to project in database)
-    $('#fields_table').on 'click', '.field_delete', ->
-      # fid of row being deleted
-      fid = $(@).closest('a').attr('fid')
-
-      # Row index of row being deleted
-      rowIndex = $(@).closest('tr').index() + 1
-
-      # Row name of row being deleted
-      rowName = $(@).closest('tr').attr('name')
-
-      # fid != 0 when the field exists in the database
-      if fid != '0'
-        hiddenDeletedFields = $('#hidden_deleted_fields')
-        if rowName == 'latitude'
-          hiddenDeletedFields.val(hiddenDeletedFields.val() + fid + ',' + (parseInt(fid) + 1) + ',')
-        else if rowName == 'longitude'
-          hiddenDeletedFields.val(hiddenDeletedFields.val() + fid + ',' + (parseInt(fid) - 1) + ',')
-        else
-          hiddenDeletedFields.val(hiddenDeletedFields.val() + fid + ',')
-        callDeleteRow(rowIndex, rowName, fid)
-      else
-        callDeleteRow(rowIndex, rowName, '')
+  if namespace.controller is 'projects' and namespace.action is 'edit_formula_fields'
+    # Add row(s), increment counters, disable add buttons (for timestamp/location fields)
+    # addRow takes input box for name, type of field, input box for units (number only) or
+    # "deg" for lat/long, input box for restrictions (text only), delete
+    $('#number').click ->
```

Comment:
We should probably use handlebars for rows instead of dumping html into the dom at some point for all the manual entry/field creation pages. 
@pat-son maybe a summer assignment for new people?